kernel/include/kswap.h

```diff
@@ -117,7 +117,7 @@ static ALWAYS_INLINE unsigned int do_swap(unsigned int key,
 		wait_for_switch(new_thread);
 		arch_switch(new_thread->switch_handle,
 			     &old_thread->switch_handle);
-		sys_trace_thread_switched_in();
+
```

Comment:
other architectures such as: x86, xtensa also need to add sys_trace_thread_switched_in() in its own arch_switch code.

Reply:
xtensa is already covered